### ISSUE TITLE
Uglifier doesn't load with Ruby 2.4.0 using The Ruby Racer JS runtime

### ISSUE THREAD

**Issue by mantas** [lautis/uglifier#105]
I'm getting following error when launch Rails application with Ruby 2.4.0

> TypeError: wrong argument type Class (expected Module)

Running irb and then requiring Uglifier into it throws the same error as well.

**Comment by Apokly** [comment]
Same for me.
Backtrace first lines :
```

/home/fabien/.rvm/gems/ruby-2.4.0/gems/therubyracer-0.12.2/lib/v8/conversion.rb:23:in `include'
/home/fabien/.rvm/gems/ruby-2.4.0/gems/therubyracer-0.12.2/lib/v8/conversion.rb:23:in `block (2 levels) in <top (required)>'
/home/fabien/.rvm/gems/ruby-2.4.0/gems/therubyracer-0.12.2/lib/v8/conversion.rb:22:in `class_eval'
/home/fabien/.rvm/gems/ruby-2.4.0/gems/therubyracer-0.12.2/lib/v8/conversion.rb:22:in `block in <top (required)>'
/home/fabien/.rvm/gems/ruby-2.4.0/gems/therubyracer-0.12.2/lib/v8/conversion.rb:21:in `each'
/home/fabien/.rvm/gems/ruby-2.4.0/gems/therubyracer-0.12.2/lib/v8/conversion.rb:21:in `<top (required)>'
```

**Comment by lautis** [comment]
Thanks for the trace @Apokly. It seems that The Ruby Racer does not yet work with Ruby 2.4, but the fix is already merged: https://github.com/cowboyd/therubyracer/commit/f2ccbad7e9e3899e8743624d9b1436500100420a

**Comment by ignisf** [comment]
Hello,

therubyracer 0.12.3 was released to rubygems with Ruby 2.4 support.
